.gitignore

```diff
@@ -81,3 +81,8 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+# General MacOS 
```

Comment:
Can you please remove the space at the end of this line?

Reply:
Shouldn't there be a space between each type to ignore?

Reply:
Newlines are ok, but there is a space at the end of the line (highlight to see it).